form.tex

```diff
@@ -0,0 +1,90 @@
+\documentclass[a4paper]{article}
+
+\usepackage[bmargin=3cm, tmargin=2cm]{geometry}
+\usepackage{graphicx}
+\usepackage{xspace}
+\usepackage[latin1]{inputenc}
+\usepackage[pdftex]{hyperref}
+\setlength\parindent{0pt}
+
+\usepackage{helvet}
+\renewcommand*\familydefault{\sfdefault} %% Only if the base font of the document is to be sans serif
+\usepackage[T1]{fontenc}
+
+\pagenumbering{gobble}
+
+%%%% Defines for form filling
+\newcommand{\signature}{%
+    Signature:\\
+    \vspace{1cm}\\
+    \rule{7cm}{1pt}\\
+    Date: \vbox{\hbox{\TextField[bordercolor=, width=6cm]{\null}}\kern2pt\hrule}
+}
+
+\begin{document}
+
+\newcommand{\compyear}{2018\xspace}
+\newcommand{\competition}{SourceBots \compyear}
+
+\begin{minipage}[c]{0.30\textwidth}
+\includegraphics[height=2cm]{fig-SourceBots}
+\end{minipage}
+\hspace{0.03\textwidth}
+\begin{minipage}[c]{0.67\textwidth}
+\begin{center}
+{\large \textbf{\competition Media Consent Release Form} \par}
+\end{center}
+\end{minipage}
+\vspace{1cm}
+
+\textbf{This is not a ticket.}\\\\ However, this form is required for entry into the \competition competition.\\
+
+\vspace{0.3cm}
+\textbf{\large Privacy Policy}\\
+
+Once signed, this form will be stored securely by SourceBots for the entire time any recordings made at \competition are used. You have the right to request for the deletion of any personal data (including images) as per the EU General Data Protection Regulation (GDPR).\\
+
+\vspace{0.3cm}
+\textbf{\large Media Consent}\\
+
+By signing this release form, you (the individual mentioned below) consent to your image, voice, biographical material, or other information being recorded in the duration of the \competition event by:
```

Comment:
> being recorded in the duration of the SourceBots 2018 event by:

This reads really oddly to me. How about:

> being recorded at the SourceBots 2018 competition by:

Or perhaps (if we want 'event' in there to clarify that it's the competition weekend):

> being recorded at the SourceBots 2018 competition event by:

I suggest that we wrapper the "SourceBots 2018 competition event" portion into an `\event` command; I don't feel particularly strongly about this part, nor how that command is constructed.

Reply:
fixed in ef25480c6114edb9c048465023d2438960e11426 and I fixed the event naming in 8e77f7f6f1836b1ab907e7ee5aeb07fc458abf8f